MAINTAINERS.md

```diff
@@ -1,18 +1,122 @@
+
+Once the work is merged in a feature branch, please make sure to delete the branch.  
+
+## Labels and Labeling 
+
+As a maintainer, part of your responsibility is triage.  
+
+By default, all new issues get the following labels:  untriage, bug.  As a maintainer, part of your responsibility is to review all incoming issues and triage them.  Our goal is to never have an issue sit for more than 2 days without a response.  
+
+
+## Releases
+
+Part of your job as a maintainer is to run releases as a Release Manaager (RM).  Here's how the process works:
+
+1. *Two weeks before release* A maintainer volunteers to be a release manager.  They initiates the process by posting a topic in the forum of the intent to release the next version (to facilitate discussion and feedback)
+2. The RM then proposes a “feature freeze” date (and intent to cut the branch).  This should be a date  *one week* in the future (and therefore one week before release).  During this week, all maintainers will: 
+	1. Mark “blocker” issues preventing a successful release
+	2. Merge all PRs that are ready into the main, or clearly label them as being for a future release.  
+	3. Review merged PRs and determine if they need a backport to the current release. 
+	4. If a maintainer is not sure if a merged PR needs a backport, they should post in the PR asking the submitter their intention. 
+3. 24 hours prior to feature freeze (cutting the branch):
+    1. RM update the forum post with a “final notice“ message to ensure all blockers are identified (some may be fixed by then).
+4. On day of feature freeze:
+    1. RM posts a “feature freeze” notice that the branching is underway and no more PRs should be merged.
+    2. RM branches from the .x branch into a new .m+1 branch. (.x is now the next minor)
+    3. RM sends a “branching complete” email that the branch has been cut and a reminder of the release date requesting all blockers be cleared in time of the release
+5. During the week between "feature freeze" and release, all maintainers will:
+	1. Will monitor issues looking for fixes that need to go in to the release. 
+5. 24 Hours prior to release:
+    1. RM sends a “final notice” for blockers email; also requesting a “speak now or forever hold your peace” on critical blockers. If something is holding up the release a postponed release may occur.
+6. On Day of Release:
+    1. RM tags the branch with the release version
+    2. RM begins building and signing artifacts
+    3. RM stages artifacts to maven repo 
+    4. an email is sent once artifacts are published requesting a quorum vote which includes committers run smoke tests on the staged artifacts;
+    5. RM sums votes 
+    6. RM publishes tested artifacts
+
+
+## Pull Requests (PRs)
+
+### PR Lifecycle
+
+Here's the life cycle of a PR:
```

Comment:
I would refer to CONTRIBUTING.md somewhere in here and try to not duplicate what the contributor is doing.

Reply:
removed PR lifecycle, added link to Contributing.